pkg/instancegroups/instancegroups.go

```diff
@@ -297,9 +295,8 @@ func (r *RollingUpdateInstanceGroup) DeleteInstance(u *cloudinstances.CloudInsta
 	}
 	if nodeName != "" {
 		glog.Infof("Stopping instance %q, node %q, in group %q (this may take a while).", id, nodeName, r.CloudGroup.HumanName)
-	} else {
-		glog.Infof("Stopping instance %q, in group %q (this may take a while).", id, r.CloudGroup.HumanName)
 	}
+	glog.Infof("Stopping instance %q, in group %q (this may take a while).", id, r.CloudGroup.HumanName)
```

Comment:
I'm not sure this change is correct. If `nodeName` is not an empty string then both this line and line 297 will log an near identical line.

Reply:
You're right, there's no early return here so we still need the `else` here .